tests/parametric/test_crashtracking.py

```diff
@@ -4,13 +4,12 @@
 import json
 import pytest
 
-from utils import bug, context, features, scenarios, logger
+from utils import bug, features, scenarios, logger
 
 
 @scenarios.parametric
 @features.crashtracking
 class Test_Crashtracking:
-    @bug(context.library >= "ruby@2.7.2-dev", reason="APMLP-335")
```

Comment:
Do you know at which version it has been fixed ? 

It'd be better to declare it here, otherwise it may break your backport for older version. For instance, if it has been fixed at `2.8.0`, the good way to declare is : 

```python
@bug("ruby@2.7.2-dev" <= context.library < "ruby@2.8.0" , reason="APMLP-335")
```